main.go

```diff
@@ -139,7 +63,11 @@ func main() {
 		log.Fatal("db.uri must be set in the configuration file")
 	}
 
-	dbconn = sqlx.MustConnect("postgres", dbURI)
+	dbconn = otelsqlx.MustConnect(
+		"postgres",
+		dbURI,
+		otelsql.WithAttributes(semconv.DBSystemPostgreSQL),
+	)
```

Comment:
This isn't even opentelemetry-related, but I've been trying to add it to things to avoid connection starvation situations -- could be good to add a `dbconn.SetMaxOpenConns` call after this. I've been using 10. In other stuff, I've also been setting a max idle time with `dbconn.SetConnMaxIdleTime` but I also think I've been setting it too low for our usage patterns. So your call there, I'd imagine.

Reply:
Thanks, should be fixed.